onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -718,10 +718,10 @@ def aten_broadcast_tensors(tensors: Sequence[TensorType]) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_broadcast_to(self: TensorType, size: INT64) -> TensorType:
+def aten_broadcast_to(self: TTensor, size: INT64) -> TTensor:
```

Comment:
Missing decorator?

On a side note, is there a way to make the CI tox log show the failure messages more nicely? (The exact message is folded on line 15, and it requires unfold + search/scroll down)

![image](https://user-images.githubusercontent.com/9376104/214128054-b7342bd2-e1c9-413e-b099-45f4ed1f7d2b.png)

Reply:
Thought that was a nice feature (by tox-gh)

Reply:
> Thought that was a nice feature (by tox-gh)

Yea it would be nice to just show the final summary of error and fold away the rest.

Reply:
I don't know if it can find just the error message. rn it folds the outputs from different test envs (py39, build) so ppl can unfold only the section of interest